src/jsTest/kotlin/it/krzeminski/snakeyaml/engine/kmp/internal/TargetUtils.kt

```diff
@@ -0,0 +1,5 @@
+package it.krzeminski.snakeyaml.engine.kmp.internal
+
+// returns 'true' is we are in the browser
+actual fun areEnvVarsSupported(): Boolean =
+    js("return (typeof window === 'undefined' || typeof document === 'undefined')") as Boolean
```

Comment:
Double-check it returns true for Nodejs.

Reply:
It does:
![Screenshot_2025-01-06-21-41-53-066_com android chrome-edit](https://github.com/user-attachments/assets/687cc4d5-5878-4d2e-981c-4fc569a1aa1f)